card/lib/cardio/seed.rb

```diff
@@ -6,15 +6,87 @@ module Seed
                 schema_migrations_deck schema_migrations_deck_cards].freeze
 
     class << self
-      attr_accessor :path, :test_path
+      def default_path
+        env = Rails.env.test? ? "test" : "production"
+        db_path env, 0
+      end
 
-      def db_path *args
-        parts = [Cardio.gem_root, "db"] + args
-        File.join(*parts)
+      def path
+        if ENV["UPDATE_SEED"]
+          args = Rails.env.test? ? ["production", 0] : ["production", 1]
+          db_path(*args)
+        else
+          default_path
+        end
+      end
```

Comment:
There's a lot of power in this little bit of code. Let me give some context.

So there are two kinds of seed yaml files:
- fixtures, which are the standard rails things that mimic data tables directly, and
- pods, or whatever we're calling our special seed representation that mimics our API

Fixtures are generally a pain to write by hand. You have to manage ids, actions, references, etc.  So there is only one place we're doing that manually for now, which is in `card/mod/core/data/fixtures` . That defines our current minimal seeding of 11 cards.

All other seed data will be generated by (a) reseeding with existing features, and then (b) eating pods to add to it. But, importantly, this process is nestable. So the production seed data in the defaults mod uses the core fixtures, the test seed data in that mod uses the production data, etc.  Wikirate seed data will use fixtures from the defaults mod, of course.

These lines of codes are really what make that whole system tick (and will probably need to get more sophisticated)